public/js/cohortStudent.js

```diff
@@ -1,7 +1,65 @@
-
-
+const addStd = document.getElementById('addStd');
+const studentName = document.getElementById('studentName');
+const gitHubUserName = document.getElementById('gitHubUserName');
+const addStudentButton = document.getElementById('addStudentButton');
+const loading = document.getElementById('loading');
+const URL = window.location.href;
+const splitUrl = URL.split('/');
+const cohortID = splitUrl[splitUrl.length - 2];
+const deleteStudentButton = document.querySelectorAll('.delete');
 
 const addStudents = () => {
   addStd.classList.toggle('sectionAddstd--visible');
 };
 
+addStudentButton.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const studentNameValue = studentName.value;
+  const gitHubUserNameValue = gitHubUserName.value;
+  if (
+    studentNameValue.trim().length !== 0 
```

Comment:
`Boolean(studentNameValue.trim()) === Boolean(studentNameValue.trim().length === 0) // true`
Meaning that you don't have to specify the `.length !== 0` part, it is self explanatory to JavaScript.

Reply:
new thing for me , thanks nour